instrumentation/src/com/intellij/rt/coverage/instrumentation/LineEnumerator.java

```diff
@@ -37,6 +45,12 @@
   private final MethodVisitor myWriterMethodVisitor;
   private final BranchDataContainer myBranchData;
 
+  private final Map<Label, Jump> myOriginalLabelToJump = new HashMap<Label, Jump>();
```

Comment:
All these extra data and processing should be enabled only by option => extract new class?